src/Etg.Yams.Core/YamsConfig.cs

```diff
@@ -88,9 +88,15 @@ public class YamsConfig
         public int ProcessWaitForExitInSeconds { get; private set; }
 
         /// <summary>
-        /// Whether a console window should be shown or not
+        /// Whether the process is started with UseShellExecute or not
         /// </summary>
-        public bool ShowApplicationProcessWindow { get; private set; }
+        [Obsolete("ShowApplicationProcessWindow is obsolete; use UseShellExecute")]
```

Comment:
This is not needed, it's sufficient to keep YamsConfigBuilder backward compatible as you did.